src/components/Callout/Callout.tsx

```diff
@@ -59,8 +62,14 @@ export class Callout extends React.Component<ICalloutProps, ICalloutState> {
   }
 
   public render() {
-    let { className, targetElement, isBeakVisible, beakStyle, children } = this.props;
+    let { className, targetElement, isBeakVisible, beakStyle, children, hasCloseButton, closeButtonAriaLabel, onDismiss  } = this.props;
     let { positions, slideDirectionalClassName } = this.state;
+
+    let closeButton;
+    if (hasCloseButton) {
+      closeButton = <Button disabled={ false } className='ms-Callout-closeButton' buttonType={ ButtonType.icon }  onClick={ onDismiss }       icon='x' title='Close' ariaLabel={ closeButtonAriaLabel } />;
```

Comment:
Feels like these props should be broken out onto their own lines for readability